feature/sflow/otg_tests/sflow_base_test/README.md

```diff
@@ -0,0 +1,67 @@
+# SFLOW-1: sFlow Configuration and Sampling
+
+## Summary
+
+Verify configuration of sflow and sFlow sample data.
+
+## Procedure
+
+*   SFLOW-1.1 Configure sFlow on DUT
+    * Configure DUT to send sflow samples to ATE port 2
```

Comment:
This seems to need ATE port2 to be setup in the packet capture mode. Should that be clarified here as well?

So the topology seems something like

ATE:port1 <-> port1:DUT:Port3 <-> port3:ATE <--- This is the Data-plane traffic flow path for which the packets are sampled and sent tot he collector simulation which is reachable on ATE:Port2

DUT:Port2 <-> Port2:ATE <<---- this ATE port2 is in the capture mode.

Reply:
So sflow-1.1 is to verify the config,  sflow-1.2 has the traffic generation and capture.  Packet capture is specified as  required in sflow-1.2.  The code (which I am also writing) has the capture configuration.